src/core-styling/mixins/_types.scss

```diff
@@ -0,0 +1,44 @@
+@use "../settings/font";
+@use "../../../node_modules/sass-rem/rem";
+
+$rem-baseline: font.$size;
+$rem-fallback: true;
```

Comment:
Can safely be `false` now.

Reply:
Would simplify things, and means we don't need the override either.